lib/block-supports/border.php

```diff
@@ -136,6 +179,54 @@ function gutenberg_apply_border_support( $block_type, $block_attributes ) {
 	return $attributes;
 }
 
+/**
+ * Generates CSS classes and longhand CSS styles for an individual side border.
+ *
+ * If some values are omitted from the border configuration, using shorthand
+ * styles would lead to `initial` values being used instead of the more
+ * desirable inherited values. This could also lead to browser inconsistencies.
+ *
+ * @param string        $side       The side the classes and styles are being generated for.
+ * @param array         $border     Array containing border color, style, and width values.
+ * @param WP_Block_type $block_type Block type.
```

Comment:
Tiny capitalisation nit:

```suggestion
 * @param WP_Block_Type $block_type Block type.
```

Looks like it also exists on the comment for `gutenberg_apply_border_support`

Reply:
Well spotted.